packages/amplify-graphql-api-construct/.jsii

```diff
@@ -3487,7 +3486,7 @@
     "stability": "stable"
   },
   "homepage": "https://github.com/aws-amplify/amplify-category-api.git",
-  "jsiiVersion": "5.2.11 (build f6c85b3)",
+  "jsiiVersion": "5.1.12 (build 0675712)",
```

Comment:
Any particular reason for this downgrade?

Reply:
It's an upgrade! I think Jsii has just updated.